consensus/polybft/contracts_initializer.go

```diff
@@ -39,6 +39,9 @@ func initHydraChain(polyBFTConfig PolyBFTConfig, transition *state.Transition) e
 		Governance:          polyBFTConfig.Governance,
 		HydraStakingAddr:    contracts.HydraStakingContract,
 		HydraDelegationAddr: contracts.HydraDelegationContract,
+		AprCalculatorAddr:   contracts.APRCalculatorContract,
+		RewardWalletAddr:    contracts.RewardWalletContract,
+		HydraVaultAddr:      contracts.FeeHandlerContract,
```

Comment:
HydraVaultAddr variable name must be changed, again, because of not give enough info about what the variable is about. (We have two instances of the HydraVault)

Reply:
Okay, but this one must be changed in the contracts.